ui/desktop/app/styles/app.scss

```diff
@@ -711,3 +711,14 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     color: var(--token-color-foreground-primary);
   }
 }
+
+// Unsupported controller modal
+.unsupported-modal-body {
+  display: grid;
+  gap: 1rem;
+}
+
+.hds-dismiss-button {
```

Comment:
Where is this used? If it's internal to HDS we're setting _all_ of them to not display now. 

Is there a reason why it doesn't disappear automatically with `isDismissDisabled` set to true? Should we even be removing this from a design perspective or is it a bug with HDS?

Reply:
Hds does not hide the button when `isDismissDisabled` is true. I originally decided to hide it to match the designs. However, I was getting a focus-trap error which is due to hiding the dismiss button. Hds docs mention it [here](https://helios.hashicorp.design/components/modal?tab=code#focus-and-focus-trap).
Due to the error, I removed this style for now.